antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/ParserRuleContext.kt

```diff
@@ -37,34 +37,45 @@ import kotlin.reflect.KClass
 @Suppress("MemberVisibilityCanBePrivate")
 public open class ParserRuleContext : RuleContext {
   public companion object {
-    @JvmStatic
+    @JvmField
     public val EMPTY: ParserRuleContext = ParserRuleContext()
   }
 
+  // This does not exist in the Java runtime
+  public val position: Position?
+    get() {
+      val start = start
```

Comment:
I would write it as `val start = this.start` and `val stop = this.stop`, to make it more obvious